--- katello.spec.orig	2013-07-26 13:20:29.279156840 -0400
+++ katello.spec	2013-07-26 13:21:28.645646764 -0400
@@ -51,6 +51,10 @@
 BuildRequires:   %{scl_ruby}
 Requires:        %{scl_ruby}
 
+Epoch: 1
+Patch1: splice_reports_1.patch
+BuildRequires: %{scl_prefix}rubygem(splice_reports)
+
 %description
 Provides a package for managing application life-cycle for Linux systems.
 
@@ -438,6 +442,10 @@
 %build
 export RAILS_ENV=build
 
+cat <<EOF > bundler.d/splice_reports.rb
+gem 'splice_reports'
+EOF
+
 #don't distribute quiet_paths
 rm -f config/initializers/quiet_paths.rb
 
@@ -453,6 +461,8 @@
     rm -rf bundler.d/assets.rb
 %endif
 
+rm bundler.d/splice_reports.rb
+
 #replace shebangs for SCL
 %if %{?scl:1}%{!?scl:0}
     find script/ -type f | xargs sed -ri '1sX(/usr/bin/ruby|/usr/bin/env ruby)X%{scl_ruby}X'
