--- katello.spec.original	2013-07-30 14:29:24.390902573 -0400
+++ katello.spec	2013-07-30 20:33:27.222340817 -0400
@@ -51,6 +51,10 @@
 BuildRequires:   %{scl_ruby}
 Requires:        %{scl_ruby}
 
+Epoch: 1
+Patch1: splice_reports_1.patch
+BuildRequires: %{scl_prefix}rubygem(splice_reports)
+
 %description
 Provides a package for managing application life-cycle for Linux systems.
 
@@ -437,6 +441,9 @@
 
 %build
 export RAILS_ENV=build
+cat <<EOF > bundler.d/splice_reports.rb
+gem 'splice_reports'
+EOF
 
 #don't distribute quiet_paths
 rm -f config/initializers/quiet_paths.rb
@@ -491,6 +498,8 @@
     mv lib/tasks_disabled lib/tasks
 %endif
 
+rm bundler.d/splice_reports.rb
+
 #man pages
 a2x -d manpage -f manpage man/katello-service.8.asciidoc
 
